[MS] Refresh the folder when a copy ended (#8972)
Scille/parsec-cloud: newsfragments/8970.bugfix.rst

> Correctly refresh the list of files after a copy ended

Reviewer: ```suggestion
The file list is not updated after copying a file
```

> Fixed the list of files not being properly updated after copying a file

Reviewer 2: ```suggestion
Fixed the file list not being properly updated after copying a file
```
